__tests__/endpoint.test.js

```diff
@@ -175,164 +175,38 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
-describe("POST /api/articles/:article_id/comments", () => {
-  test("Posts a new comment into an article with a given ID", () => {
-    const newComment = {
-      username: "rogersop",
-      body: "Hello this is my new comment",
-    };
-    return request(app)
-      .post("/api/articles/1/comments")
-      .send(newComment)
-      .expect(201)
-      .then(({ body }) => {
-        expect(body).toMatchObject({
-          comment_id: 19,
-          body: "Hello this is my new comment",
-          author: "rogersop",
-        });
-      });
-  });
 
-  test("404: If a new comment is posted with a username that doesn't exist an error will be returned", () => {
-    const newComment = {
-      username: "TomdaBomb",
-      body: "Hello this is my new comment",
-    };
+describe("DELETE comment by ID", () => {
+  test("responds with an empty object if no content after a successful deletion", () => {
     return request(app)
-      .post("/api/articles/1/comments")
-      .send(newComment)
-      .expect(404)
+      .delete("/api/comments/3")
+      .expect(204)
       .then(({ body }) => {
-        expect(body.msg).toBe("No username found");
+        expect(body).toEqual({});
       });
   });
-
-  test("400 returns an error if a comment is not complete", () => {
-    const newComment = {
-      username: "rogersop",
-    };
-    return request(app)
-      .post("/api/articles/1/comments")
-      .send(newComment)
-      .expect(400)
-      .then(({ body }) => {
-        expect(body.msg).toBe("bad request");
-      });
-  });
-
-  test("404 given a valid but empty article ID the endpoint will respond with an error", () => {
+  test("404 reponse an error if trying to delete on path that that does not exist ", () => {
     return request(app)
-      .get("/api/articles/555667/comments")
+      .delete("/api/not-a-comment/3")
```

Comment:
This is a general 404 test really - the url does not exist aka not found. You presumably already have a test for this and it it is not relevant to this URL. Everything - happy and sad paths - for this endpoint is covered by the other tests so it can be removed 👍 

And the msg for this error should be "not found" or similar not "bad request". Basically the url here is not found.